docs/Contributors_Guide/github_workflow.rst

```diff
@@ -22,6 +22,269 @@ develop branch.  When a release candidate for METplus has been determined,
+        each repository.
+
+      - Scroll down to the **Danger zone** and change **Visibilty** from its default value of **Private**
+        to **Public**.
+
+   - Select **Manage access** on the left hand navigation bar.
+
+      - By default, the project creator has **Admin** access.
+
+      - Add **Admin** access for any user with that level of access to any one
+        of the repositories managed by this project.
+
+      - Add **Write** access for the **METplus** group.
+
+   - Locate the **Custom fields** section in the left hand navigation bar
+
+      - Select **Status** and retain the default list of options, but delete the
+        **New** option by clicking the **X** to its right.
+
+      - Select **Iteration** and modify the **Field name** to be **Cycle**.  Delete any existing cycles.
+        Under **More options** select a **Start on** date and set the default **Duration** as 6 weeks.
+        Click the **Add** and **Add iteration** buttons to create 5 cycles, each with the default duration
+        of 6 weeks. Modify the cycle names to be **Beta1**, **Beta2**, and so on. Click **Save changes**.
+
+      - Select **Estimate** and click the three dots to the right and **Delete field**.
+
+   - Click the back arrow to return to the project page.
+
+3. Update the project **Workflows**.
+
+   - Click on the three dots to the right of the project name to see **More options**
+     and select **Workflows**.  Modify these settings as follows.
+
+      - Enable the **Item added to project** workflow and set the status to **Backlog**.
+
+      - Enable the **Item reopened** workflow and set the status to **In progress**.
+
+      - Enable the **Item closed** workflow and set the status to **Done**.
+
+      - Enable the **Pull request merged** workflow and set the status to **Done**.
+
+      - Leave all other workflows disabled.
+
+   - Click the back arrow to return to the project page.
+
+4. Create project **Views**.
+
+   - Each view appears as a tab on the project page.  Create a new view as described below.
+
+      - Select the **+ New view** option.
+
+      - Click on the view name to modify it, rename it as **All Cycles**, and hit **Enter**.
+
+      - Click on the down arrow and, under **Configuration**, select the **Fields** option. Enable
+        the options for **Title**, **Repository**, **Assignees**, **Cycle**, **Status**,
+        **Linked Pull Request**, and **Reviewers**. Drag and drop the items to reorder them as listed above.
+
+      - In the resulting view, click the three dots in the **Status** column. Select the **Sort descending**
+        and **Group by values** options.
+
+      - The blue dot on the down arrow for this tab indicates that there are unsaved changes.
+        Select the **Save changes** option.
+
+   - Click on the down arrow and select **Duplicate View**.  Do this 7 times and name/refine these views as follows.
```

Comment:
If the suggestion below this one is applied, then the number here should be updated.

```suggestion
   - Click on the down arrow and select **Duplicate View**.  Do this 8 times and name/refine these views as follows.
```